packages/e2e/package.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
The indent spacing here isn't correct. We're not linting json, so it has to be manually controlled at this point.